specs/preinstalls.md

```diff
@@ -0,0 +1,206 @@
+# Preinstalls
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [create2Deployer](#create2deployer)
+- [Safe](#safe)
+- [SafeL2](#safel2)
+- [MultiSend](#multisend)
+- [MultiSendCallOnly](#multisendcallonly)
+- [SafeSingletonFactory](#safesingletonfactory)
+- [MultiCall3](#multicall3)
+- [Arachnid's Deterministic Deployment Proxy](#arachnids-deterministic-deployment-proxy)
+- [Permit2](#permit2)
+- [ERC-4337 EntryPoint](#erc-4337-entrypoint)
+- [ERC-4337 SenderCreator](#erc-4337-sendercreator)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+[Preinstalled smart contracts](./glossary.md#preinstalled-contract-preinstall) exist on Optimism
+at predetermined addresses in the genesis state. They are  similar to precompiles but instead run
+directly in the EVM instead of running  native code outside of the EVM.
+
+These preinstalls are commonly deployed smart contracts that are being placed as genesis for convenience.
+It's important to note that these contracts do not have the same security guarantees
+as [Predeployed smart contracts](./glossary.md#predpeloyed-contract-predeploy).
+
+The following table includes each of the preinstalls. The system version
+indicates when the preinstall was introduced. The possible values are
+`Bedrock` or `Canyon`. Deprecated contracts should not be used.
+
+| Name                                      | Address                                    | Introduced  | Deprecated | Proxied |
```

Comment:
There is no longer a concept of "introduced", "deprecated" and "proxied"

The `create2deployer` is not a preinstall as its added as part of the canyon hardfork, it is part of the EL consensus as an irregular state transition. You can remove it from this list